helm/overlay/kustomization.yaml

```diff
@@ -22,3 +22,23 @@ patches:
     target:
       kind: MutatingWebhookConfiguration
       name: "(mutating-webhook-configuration)"
+#  - patch: |
```

Comment:
can we just remove this instead of commenting it out? do we need it again in the future?

Reply:
will remove